services/web/src/stores/session.js

```diff
@@ -87,17 +90,6 @@ export class SessionProvider extends React.PureComponent {
     }
   };
 
-  // TODO: this shouldn't be needed
-  reloadUser = async () => {
-    const { data } = await request({
-      method: 'GET',
-      path: '/1/users/me',
-    });
-    this.setState({
-      user: data,
-    });
-  };
-
```

Comment:
I don't think bootstap really can replace just reloading the user.

The user case where is that you modify e.g. the user.name or some setting, that has no impact on anything but the user. 

Bootstrap semantic is very different from reloadUser.

Reply:
What we want to avoid is duplicate fetching logic. In either case there is no need to refresh the data from the server as you have just pushed the updates. I think we need something more like `updateUser` that simply updates the session state